wled00/FX.cpp

```diff
@@ -4861,6 +4861,49 @@ static const char _data_FX_MODE_FLOWSTRIPE[] PROGMEM = "Flow Stripe@Hue speed,Ef
 #define XY(x,y) SEGMENT.XY(x,y)
 
 
+#ifdef WLED_USE_AA_PIXELS
+/////////////////////////////
+//    2D Analog Clock      //
+/////////////////////////////
+uint16_t mode_2DAnalogClock(void) {                  // By Andras Fekete (bandi13)
+  if (!strip.isMatrix) return mode_static(); // not a 2D set-up
+
+  const uint16_t cols = SEGMENT.virtualWidth();
+  const uint16_t rows = SEGMENT.virtualHeight();
+  const uint16_t centerX = cols >> 1;
+  const uint16_t centerY = rows >> 1;
+  const uint16_t radius = min(centerX, centerY);
+
+  auto hours_in_day = (localTime / (60 * 60)) % 12;
```

Comment:
Please use `time_t` instead of `auto`

Reply:
Ouch! @willmmiles will spank you. 😉 We are too old for this auto madness.

Reply:
Hehe! :) I'm a big AAA fan - few things annoy me more than having to remind the compiler a type name it already f-ing knows - but I'll also be the first to say that *consistency in the local code style* is much more important.